### DISCUSSION TITLE
hiyoriUI.py启动无报错，但打开的网页是完全空白的，然而http://127.0.0.1:5000/docs是能打开的  该如何处理呢？

### DISCUSSION THREAD

**Discussion by goldenbubble2023** [fishaudio/Bert-VITS2#274] [Q&A] [answered]
**终端信息如下：**
F:\GPTS\Bert-VITS2-2.3>F:\GPTS\Bert-VITS2-2.3\venv\python hiyoriUI.py
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
01-12 00:15:47 INFO     | hiyoriUI.py:163 | 开始挂载网页页面
01-12 00:15:47 INFO     | hiyoriUI.py:180 | 开始加载模型
01-12 00:15:47 WARNING  | hiyoriUI.py:719 | 本地服务，请勿将服务端口暴露于外网
01-12 00:15:47 INFO     | hiyoriUI.py:720 | api文档地址 http://127.0.0.1:5000/docs

**Comment by Stardust-minus** [comment]
> **终端信息如下：**
> F:\GPTS\Bert-VITS2-2.3>F:\GPTS\Bert-VITS2-2.3\venv\python hiyoriUI.py
> Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
> Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
> Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
> 01-12 00:15:47 INFO     | hiyoriUI.py:163 | 开始挂载网页页面
> 01-12 00:15:47 INFO     | hiyoriUI.py:180 | 开始加载模型
> 01-12 00:15:47 WARNING  | hiyoriUI.py:719 | 本地服务，请勿将服务端口暴露于外网
> 01-12 00:15:47 INFO     | hiyoriUI.py:720 | api文档地址 http://127.0.0.1:5000/docs
> 

@jiangyuxiaoxiao 看一眼？

**Comment by jiangyuxiaoxiao** [comment, marked as answer]
参考https://github.com/jiangyuxiaoxiao/Bert-VITS2-UI/issues/4
